### PR TITLE
PHP 8 supporting images

### PR DESCRIPTION
**Pull request by g-foster2020**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -39,6 +39,24 @@ services:
...
         BASEOS: bullseye
 
+  php80-fpm-buster-base:
+    image: my127/akeneo:8.0-fpm-buster${TAG_SUFFIX:-}
+    build:
+      context: ./
+      dockerfile: base.Dockerfile
+      args:
+        VERSION: "8.0"
+        BASEOS: buster
```
Comment (andytson-inviqa) [inline docker-compose.yml]: lets not bother with buster till we have a real reason to use it over the more recent bullseye

Comment (andytson-inviqa) [reply docker-compose.yml]: each image build adds additional time, so we don't want to increase that for unnecessary images

Comment (andytson-inviqa) [inline docker-compose.yml]:
```suggestion
        BASEOS: bullseye
```

Comment (g-foster2020) [reply docker-compose.yml]: Understood

**Review by andytson-inviqa — DISMISSED** [review]

**Review by andytson-inviqa — APPROVED** [review]

**Pull request merged**